Getting-Started/index-v8.md

```diff
@@ -0,0 +1,130 @@
+---
+
+# Getting Started with Umbraco 8
+*All the basics of using Umbraco v8. From making a local installation to extending the backend.*
+
+The newest major version of Umbraco comes with a lot of new big features, as well as a long list of general improvements to the CMS in general.
+
+:::note
+This is a work in progress. 
+
+Want to help out making documentation for V8? Please check out our [Contribution guidelines](../Contribute).
+:::
+
+## [What's new in Umbraco 8?](New-in-V8.md)
+
+Learn more about the new features in Umbraco 8.
+
+## Umbraco 8 Documentation
+
+<div class="row">
+	<div class="col-xs-3">
+		<span class="dot big icon-Palette">
+			<span class="line v-line top"></span>
+			<span class="line v-line"></span>
+			<span class="line h-line"></span>
+		</span>
+		<span class="dot small">
+			<span class="line v-line"></span>
+			<span class="line h-line"></span>
+		</span>
+	</div>
+	<div class="col-xs-9">
+		<div class="row explain">
+			<div class="col-xs-12">
+				<h4 class="text-right">Design</h4>
+				<small>Create templates and views with your own markup and use Razor to render content</small>
+			</div>
+			<div class="col-sm-6">
+				<h5><a href="Design/Rendering-Content/index-v8.md">Rendering content</a></h5>
+				<small>Render the values of the current page or the result of a query against the content cache.</small>
+			</div>
+		</div>
+	</div>
+</div>
+
+
+<div class="row">
+	<div class="col-xs-3">
+		<span class="dot big icon-Code">
+			<span class="line v-line top"></span>
+			<span class="line v-line"></span>
+			<span class="line h-line"></span>
+		</span>
+		<span class="dot small">
+			<span class="line v-line"></span>
+			<span class="line h-line"></span>
+		</span>
+	</div>
+	<div class="col-xs-9">
+		<div class="row explain">
+			<div class="col-xs-12">
+				<h4 class="text-right">Code</h4>
+			</div>
+			<div class="col-sm-6">
+				<h5><a href="Code/Debugging/Logging/index-v8.md">Logging</a></h5>
+				<small>Understanding how logging works, and learn to setup custom logging.</small>
+			</div>
+		</div>
+	</div>
+</div>
+
+
+<div class="row">
+	<div class="col-xs-3">
+		<span class="dot big  icon-untitled">
+			<span class="line v-line top"></span>
+			<span class="line v-line"></span>
+			<span class="line h-line"></span>
+		</span>
+		<span class="dot small">
+			<span class="line v-line"></span>
+			<span class="line h-line"></span>
+		</span>
+	</div>
+	<div class="col-xs-9">
+		<div class="row explain">
+			<div class="col-xs-12">
+				<h4 class="text-right"><a href="../Extending/index-v8.md">Extending</a></h4>
+				<small>Using the backoffice to create data</small>
+			</div>
+			<div class="col-sm-6">
+				<h5><a href="../Extending/Content-Apps/index-v8.md">Content Apps</a></h5>
+				<small>Learn how to develop content apps / tools for your editors.</small>
+			</div>
+			<div class="col-sm-6">
```

Comment:
The only thing that changed with tree actions is that one thing changed names because there was a typo in v7.. Not sure it should be added here as "What's new".. 😉